### PR TITLE
1470 unable to enrol into testing program 

### LINKED ISSUE #1470: Unable to enrol into testing program
Can't enroll a patient into the testing program (most likely because there is no control for the now require program status).

Possible solution: make the program status optional and interpret a missing status as 'Active'?  (to be discussed in next catchup)

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #1470 

# 👩🏻‍💻 What does this PR do? 
Defaults all program enrolment statuses to `Active`

# 🧪 How has/should this change been tested? 
See any enrolment form

### CODE REVIEW

File: server/service/src/programs/schemas/program_enrolment.json [hunk 1]
```diff
@@ -1,13 +1,42 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
+    "Note": {
```
Comment (roxy-dao) [inline server/service/src/programs/schemas/program_enrolment.json:4]: Not sure why this was still missing after the note PRs were merged 🤔

File: client/packages/programs/src/hooks/usePatientCreateStore.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import create from 'zustand';
+import { create } from 'zustand';
```
Comment (roxy-dao) [inline client/packages/programs/src/hooks/usePatientCreateStore.ts:1]: Off track - depreciated

File: server/service/src/sync/init_programs_data.rs [hunk 3]
```diff
@@ -433,6 +433,7 @@ fn program_1() -> SchemaProgramEnrolment {
         enrolment_datetime: Utc::now().to_rfc3339(),
         program_enrolment_id: Some("programEnrolmentId1".to_string()),
         status: ProgramEnrolmentStatus::Active,
+        notes: Default::default(),
```
Comment (roxy-dao) [inline server/service/src/sync/init_programs_data.rs:436]: Again - missing from one of the note PRs

**Review by clemens-msupply — APPROVED** [review]
Thanks! not sure about the source of the note merge problems...

**Pull request merged**